tools/ovc/unit_tests/moc_tf_fe/conversion_basic_models_test.py

```diff
@@ -169,20 +60,21 @@ def test_cutting_fp32(self, input_freezing_value, inputs, expected,
 
     @generate(
         *[
-            (
-                    "x[1,4],y[4]",
-                    {"x": np.array([[3, 2, 1, 5]], dtype=np.int32), "y": np.array([0, -1, -7, 8], dtype=np.int32)},
-                    np.array([[3, 1, -6, 13]], dtype=np.int32),
-                    np.int32,
-                    None
-            ),
-            (
-                    "x,y",
-                    {"x": np.array([[-3, 20, 1]], dtype=np.int32), "y": np.array([[10, -11, -17]], dtype=np.int32)},
-                    np.array([[7, 9, -16]], dtype=np.int32),
-                    np.int32,
-                    None
-            ),
+            # TODO: Return this test when new 'cut' helper is introduced
+            # (
+            #         "x[1,4],y[4]",
```

Comment:
why do we need cut helper if there is no cutting?

Reply:
No cutting is temporal behavior. In future we plan to add output/input cut using new `cut` helper.

Reply:
I am afraid that we have no test remained with real inputs and outpus specified. Looks only OOB conversion without real inputs and output specification

Reply:
there is no cutting in this test and this test should work via ovc

Reply:
I checked the model, it has `tf.placeholder()` with name `x` and `tf.placeholder_with_default()` with name `y`. `tf.placeholder_with_default()` becomes constant after `input_model = moc_front_end.load(...)`. 
`input_model.get_inputs()` returns only one input with name 'x'.
The further logic applies usual input cut.

It looks like, to make this case work we need to check inputs of original TF model. But in this case the code will be FW specific and we will need to add this logic for each FW.